panel-app/src/components/analytics.ts

```diff
@@ -0,0 +1,14 @@
+import * as amplitude from '@amplitude/analytics-browser';
+
+export function initializeAmplitude() {
+	let amplitudeKey = null;
+	try {
+		amplitudeKey = process.env.AMPLITUDE_KEY;
+		amplitude.init(amplitudeKey, { defaultTracking: false });
+	} catch (e) {}
+	console.log(amplitudeKey);
```

Comment:
Don't need to console.log the key anymore, it was just to make sure it was loaded properly